src/background.js

```diff
@@ -18,119 +20,23 @@ console.log('Settings thirdparty environment variables:', thirdparty_env);
 
 var mainWindow, editWindow, reportWindow;
 
-var setApplicationMenu = function () {
-    // We clone editMenuTemplate object so we can append to submenu
-    var menus = [JSON.parse(JSON.stringify(editMenuTemplate))];
-
-    if (!editWindow) {
-        menus[0].submenu.push({
-            label: 'Deselect All',
-            accelerator: 'CmdOrCtrl+D',
-            click() {
-              BrowserWindow.getFocusedWindow().webContents.executeJavaScript("document.dispatchEvent(new CustomEvent('deselectAll'));");
-            }
-        });
-        menus[0].submenu.push({
-            role: 'delete'
-        });
-    }
+var setApplicationMenu = function() {
+    var menus = [JSON.parse(JSON.stringify(appMenuTemplate)), JSON.parse(JSON.stringify(editMenuTemplate)), JSON.parse(JSON.stringify(windowMenuTemplate))];
 
+    // Developer menu
     if (env.name !== 'production') {
         menus.push(devMenuTemplate);
     }
 
-    if (process.platform === 'darwin') {
```

Comment:
That's why we use [roles](https://github.com/electron/electron/blob/master/lib/browser/api/menu-item-roles.js).